src/Microsoft.VisualStudio.Editors/PropPages/xlf/compileproppage2.es.xlf

```diff
@@ -44,7 +44,7 @@
       </trans-unit>
       <trans-unit id="OptionStrictLabel.Text">
         <source>Option &amp;strict:</source>
-        <target state="translated">Opción &amp;strict:</target>
+        <target state="translated">Option &amp;strict:</target>
```

Comment:
Why was this untranslated?

Reply:
Linguist decided to keep terms in ENU also for other 
Option In&fer:
Option C&ompare: etc
as they are compile options

Reply:
Yet they are localized here: https://github.com/dotnet/project-system/pull/5682#discussion_r352279535?

Reply:
Each language decided how to treat them.
In bug https://devdiv.visualstudio.com/DevDiv/_workitems/edit/1021179